homeassistant/components/system_bridge/sensor.py

```diff
@@ -2,39 +2,237 @@
 from __future__ import annotations
 
 from datetime import datetime, timedelta
-from typing import Any
+from typing import Final, cast
 
 from systembridge import Bridge
 
-from homeassistant.components.sensor import SensorEntity
+from homeassistant.components.sensor import STATE_CLASS_MEASUREMENT, SensorEntity
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
     DATA_GIGABYTES,
     DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_POWER,
     DEVICE_CLASS_TEMPERATURE,
     DEVICE_CLASS_TIMESTAMP,
     DEVICE_CLASS_VOLTAGE,
     ELECTRIC_POTENTIAL_VOLT,
     FREQUENCY_GIGAHERTZ,
+    FREQUENCY_MEGAHERTZ,
     PERCENTAGE,
+    POWER_WATT,
     TEMP_CELSIUS,
 )
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import StateType
 
 from . import SystemBridgeDeviceEntity
 from .const import DOMAIN
 from .coordinator import SystemBridgeDataUpdateCoordinator
+from .model import SystemBridgeSensorEntityDescription
+
+ATTR_AVAILABLE: Final = "available"
+ATTR_FILESYSTEM: Final = "filesystem"
+ATTR_MOUNT: Final = "mount"
+ATTR_SIZE: Final = "size"
+ATTR_TYPE: Final = "type"
+ATTR_USED: Final = "used"
+
+
+BASE_SENSOR_TYPES: tuple[SystemBridgeSensorEntityDescription, ...] = (
+    SystemBridgeSensorEntityDescription(
+        key="bios_version",
+        name="BIOS Version",
+        entity_registry_enabled_default=False,
+        device_class=None,
```

Comment:
Please only set the attributes where we diverge from the default value, eg `None` in this case.